sdk/js/src/service/users.js

```diff
@@ -100,7 +100,17 @@ class Users {
         field_mask: Marshaler.fieldMask(mask),
       },
     )
-    return Marshaler.unwrapUser(response)
+
+    const result = Marshaler.unwrapUser(response)
+
+    // Get new profile picture value if the old one was deleted
+    if (mask.includes('profile_picture') || mask.includes('primary_email_address')) {
```

Comment:
In case of gravatar, the PP is attached to the `primary_email_address`, so if that changed, the profile picture changes as well. I will make the comment a bit clearer wrt this.